### ISSUE TITLE
Woocommerce 3.0

### ISSUE THREAD

**Issue by shawnvdb** [XiaoFaye/WooCommerce.NET#107]
Is there any issues with Woocommerce.Net and Woocommerce 3.0?

**Comment by XiaoFaye** [comment]
The Woocommerce.NET has not been updated to support Woocommerce 3.0 yet. My plan is to have an pre-release in early May.

**Comment by goguvi** [comment]
Despite of this you should be able to continue using the v1 of the api without problems.
The functionality already implemented by WooCommerce.Net should work correctly.

**Comment by shawnvdb** [comment]
So I upgraded to Woo 3.0 and the products work fine but the orders does not.

wc.GetOrders

Additional information: There was an error deserializing the object of type System.Collections.Generic.List`1[[WooCommerceNET.WooCommerce.Order, WooCommerceNET, Version=0.5.6.0, Culture=neutral, PublicKeyToken=null]]. The value 'total' cannot be parsed as the type 'Int32'.

**Comment by goguvi** [comment]
I have the same error.
I think the problem is on shipping lines taxes.
After update to 3.0.3 I get:
 ```
       "taxes": [
          {
            "id": "total",
            "total": []
          }
        ]
```
But on a 2.6.4 the result looks like:
```
"taxes": [
          {
            "id": 1,
            "total": "10.5"
          },
          {
            "id": 2,
            "total": ""
          }
        ]
```

**Comment by goguvi** [comment]
WooCommerce team released version 3.0.4 with shipping_lines fixed

**Comment by shawnvdb** [comment]
I upgraded to 3.0.5 and the problem is back again.

Exception thrown: 'System.Runtime.Serialization.SerializationException' in System.Runtime.Serialization.dll

Additional information: There was an error deserializing the object of type System.Collections.Generic.List`1[[WooCommerceNET.WooCommerce.Order, WooCommerceNET, Version=0.5.6.0, Culture=neutral, PublicKeyToken=null]]. The value 'total' cannot be parsed as the type 'Int32'.

I there anything I can change in the code that will fix this problem?
